docs/api/apiv3/components/schemas/project_phase_definition_collection_model.yml

```diff
@@ -0,0 +1,47 @@
+# Schema: ProjectPhaseDefinitionCollectionModel
+---
+allOf:
+  - $ref: './collection_model.yml'
+  - type: object
+    required:
+      - _links
+      - _embedded
+    properties:
+      _links:
+        type: object
+        required:
+          - self
+        properties:
+          self:
+            allOf:
+              - $ref: "./link.yml"
+              - description: |-
+                  This file links collection
+                  
+                  **Resource**: ProjectPhaseDefinitionCollectionModel
+      _embedded:
+        type: object
+        required:
+          - elements
+        properties:
+          elements:
+            type: array
+            items:
+              $ref: './project_phase_definition_model.yml'
+example:
+  _type: Collection
+  total: 2
+  count: 2
+  _links:
+    self:
+      href: '/api/v3/project_phase_definitions'
+  _embedded:
+    elements:
+      - id: 14
+        name: "Initiating"
+        _type: ProjectPhaseDefinition
+        _hint: project phase definition resource shortened for brevity
+      - id: 15
+        name: "Executing"
+        _type: ProjectPhaseDefinition
+        _hint: project phase definition resource shortened for brevity
```

Comment:
This collection representer is a `OffsetPaginatedCollection` representer and therefore additionally has the:
* pageSize
* offset

properties and the

* changeSize
* jumpTo

links.

I guess there are two possibilities for addressing this:
* Add the missing properties to the documentation
* Turn the resource into an `UnpaginatedCollection`

Given the expected smaller set of definitions per instance, using an `UnpaginatedCollection` is an option.

Reply:
Thank you, good catch!

Hmm, I'd prefer to stay future-proof by using a paginated collection right away. That way, we spare clients from having to update, even if the number of phase definitions rapidly grows unexpectedly in the future.

I noticed that I could just use the schema for `OffsetPaginatedCollection`. This means that the above links and properties should now be included as expected 👍

Reply:
That was easy then, @EinLama . The only thing missing now is for the example to also have those links.